addons/website_slides/models/slide_slide.py

```diff
@@ -396,14 +396,19 @@ def _compute_slides_statistics_category(self, read_group_res):
         """ Compute statistics based on all existing slide categories """
         slide_categories = self.env['slide.slide']._fields['slide_category'].get_values(self.env)
         keys = ['nbr_%s' % slide_category for slide_category in slide_categories]
-        result = dict((cid, dict((key, 0) for key in keys + ['total_slides'])) for cid in self.ids)
+        result = dict((cid, dict((key, 0) for key in keys + ['total_slides', 'completion_time'])) for cid in self.ids)
```

Comment:
Hey, I changed things around, and figured out how to update completion time to include  unpublished content for the users who have a right to see said content and publish them (the way LNA wanted before) by calling a compute_completion_time for a category whenever said category data is getting prepared to be rendered in a page. 
Do you think this is a right way of solving the issue?

Reply:
Just realized... Since we are not including unpublished content for the users in slide count and completion time values anymore, there are no more non-stored fields in that computed method, we are storing all the fields into database :sweat: